tests/functional/sign_in_token_code.js

```diff
@@ -40,7 +40,7 @@ const testAtOAuthApp = thenify(function () {
 
 registerSuite('signin token code', {
   beforeEach: function () {
-    email = TestHelpers.createEmail();
+    email = TestHelpers.createEmail('sync{id}');
```

Comment:
It seems a little strange to me to use a sync address for this test, makes me wonder whether we should add `tokenCode` to the list of email addresses that are forced through signin confirmation.

Reply:
We can update it, but not sure if it necessary. Sync emails require some verification, which the tests below specify as codes. Probably a better name might be `requireVerification` or something similar.

Reply:
> Probably a better name might be requireVerification or something similar.

To future proof ourselves, we should use a name that indicates the type of verification required (TOTP vs code, etc), `sync` was not a great choice in hindsight, `requireVerification` is better but doesn't go quite far enough IMO.